sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/throughputControl/ThroughputRequestThrottler.java

```diff
@@ -126,6 +134,21 @@ private void updateAvailableThroughput() {
         }
     }
 
+    private static boolean isBulkRequest(RxDocumentServiceRequest request) {
+        if (request.getOperationType() != OperationType.Batch ||
+            request.getResourceType() != ResourceType.Document) {
+
+            return false;
+        }
+
+        String isAtomicBatch = request.getHeaders().get(HttpConstants.HttpHeaders.IS_BATCH_ATOMIC);
+        if(StringUtils.isEmpty(isAtomicBatch)) {
+            return true;
+        } else {
+            return !isAtomicBatch.equalsIgnoreCase(Boolean.TRUE.toString());
```

Comment:
probably can be combined into one: StringUtils.equalsIgnoreCase

Reply:
No - isAtomicBatch might be null - I could change it to 
return StringUtils.isEmpty(isAtomicBatch) || !isAtomicBatch.equalsIgnoreCase(Boolean.TRUE.toString())

But doubt that makes it much more readable. And the pattern used here - was used in similar places as well.